modules/ve-cx/init/ve.init.mw.CXTarget.js

```
@@ -1103,11 +1113,21 @@ ve.init.mw.CXTarget.prototype.changeContentSource = function (
  *
  * @param {number} sectionNumber
  */
-ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber ) {
+ve.init.mw.CXTarget.prototype.prefetchTranslationForSection = function ( sectionNumber, set_it ) {
 	const $section = this.sourceSurface.$element.find( '#cxSourceSection' + sectionNumber );
 	if ( $section.length ) {
 		this.MTManager.getPreferredProvider().then( ( provider ) => {
 			this.translateSection( $section.prop( 'id' ), provider );
+			// auto translation
+			if (set_it) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)